packages/pigeon/lib/generator_tools.dart

```diff
@@ -432,3 +432,39 @@ Iterable<EnumeratedClass> getCodecClasses(Api api, Root root) sync* {
 /// Returns true if the [TypeDeclaration] represents an enum.
 bool isEnum(Root root, TypeDeclaration type) =>
     root.enums.map((Enum e) => e.name).contains(type.baseName);
+
+/// Formats documentation comments and adds them to current Indent.
+void addDocumentationComments(Indent indent, List<String> comments, String open,
```

Comment:
nit: this would be easier to follow with named parameters since sometimes one of the arguments is `null` for example.

Reply:
Good point